tests/pkcli/fmt_data/file1_expect.py

```diff
@@ -0,0 +1,21 @@
+def bad_foo3(
```

Comment:
Tests serve as examples and documentation. In this case, it would be nice to see what the formatter will do with different cases instead of the same one.

Reply:
@robnagler made different cases lmk if more things you would like to see black format